### PR TITLE
fix: lower read length in bytes for calculation of collateralization

### PR DESCRIPTION
**Pull request by philipdesmedt**

### CODE REVIEW

File: clarity/contracts/arkadiko-collateral-types-v1-1.clar [hunk 1]
```diff
@@ -112,6 +112,7 @@
   )
 )
 
+;; TODO - this is not in use
```
Comment (nieldeckx) [inline clarity/contracts/arkadiko-collateral-types-v1-1.clar:115]: Can we remove this method? If governance wants to update the collateral types, an updated contract can be pushed I guess? 

Or we could keep this method for now and make sure the dao-owner can call this so we can quickly update if needed.

File: clarity/contracts/arkadiko-freddie-v1-1.clar [hunk 2]
```diff
@@ -119,7 +119,12 @@
 ;; @param coll-type; contract that contains the collateral types that can be used for a vault
 ;; @param oracle; the oracle implementation that provides the on-chain price
 ;; @post uint; returns the collateralization ratio
-(define-public (calculate-current-collateral-to-debt-ratio (vault-id uint) (coll-type <collateral-types-trait>) (oracle <oracle-trait>))
+(define-public (calculate-current-collateral-to-debt-ratio
+  (vault-id uint)
+  (coll-type <collateral-types-trait>)
+  (oracle <oracle-trait>)
+  (include-stability-fees bool)
```
Comment (nieldeckx) [inline clarity/contracts/arkadiko-freddie-v1-1.clar:126]: REQ: need to update method comments

**Review by nieldeckx — CHANGES_REQUESTED** [review]
Can you explain the problem, and why the extra parameter fixes it?

**Comment by philipdesmedt** [comment]
@nieldeckx yes - basically there is a read length in number of bytes (100K bytes if I recall correctly) per read-only method through RPC.

Due to the addition of extra vault values (the pox settings I think), we went over 100K bytes read in the read only method... so I needed to fix this somehow. The easiest was to not ask for the stability fee calculation from the front-end when requesting the collateralization ratio to show in the UI.. but still use it to calculate liquidations in the liquidator. Makes sense?

File: clarity/contracts/arkadiko-collateral-types-v1-1.clar (hunk 1 above)
Comment (philipdesmedt) [reply clarity/contracts/arkadiko-collateral-types-v1-1.clar:115]: @nieldeckx what should we do? there's also the `change-risk-parameters` method that can be called by the DAO keys...

**Comment by philipdesmedt** [comment]
@nieldeckx going to merge this - we can resolve the TODO in a later PR since it is not related.

**Pull request merged**